index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>DOGTOPIA</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+      <article class="wrapper">
+
+        <div id="topBar">
+            <p><a href="https://www.rover.com/blog/great-dane-personality/">A DANE'S BLOG</a></p>
+            <ul>
+              <li>SEARCH</li>
+              <li><a href="www.http://www.akc.org/dog-breeds/great-dane/">ABOUT</a></li>
+            </ul>
+                  <h1>DOG DAYS </h1>
+                  <p>An Ironyard Academy Project Fall 2015</p>
+        </div>
+
+        <div id="main">
+            <h2>FOUNDERS OF THE CANIN-ERICA</h2>
+              <p><img src="images/Scoobydoo.jpg"/>Scooby Doo on Thursday July 16th</p>
+
+               <p>Puppy kitty ipsum dolor sit good dog lazy cat pet gate twine dinnertime string smooshy crate heel vitamins chirp. Aquatic run throw licks cockatiel bedding wag tail vaccine string behavior. Pet Gate slobber Spike mouse groom park Snowball tongue food good boy running parrot treats nap finch tooth lazy cat. Stripes slobbery birds window small animals maine coon cat vaccine yawn tabby pet supplies teeth lazy cat groom bark vitamins. Tail right paw bed bark tooth feathers house train. Bird left paw cage lick Fido leash collar mouse gimme five tongue. Litter food maine coon cat wet nose dog house crate scratcher small animals nest fetch bed string water dog. Bird Seed lazy dog wag tail yawn play dead groom maine coon cat furry ferret treats. Play Dead feeder pet supplies purr slobbery fur meow tongue heel walk.</p>
+
+               <p>Fluffy fur biscuit food dragging collar run window stay throw Buddy. Nest throw polydactyl dog house grooming wag tail kitty. Pet Gate furry cage walk paws tabby. Litter feeder toy chow kisses house train behavior barky meow lol catz parakeet polydactyl lick fluffy. Gimme Five twine hamster vaccine dog house vaccine leash collar fetch Buddy yawn slobbery pet gate meow shake biscuit stick kibble. Crate kisses warm vaccination canary head chew food. Bark roll over crate groom furry ferret maine coon cat aquatic barky right paw play dead dog stripes tuxedo kibble canary pet gate play dead wagging.</p>
+
+              <br><h4>J.D. BARK-A-FELLA</h4><br>
```

Comment:
Likely don't need the `<br>`'s. If you're wanting more space around the `<h4>`, use margin or padding in the CSS.